src/main/java/uk/gov/pay/adminusers/validations/RequestValidations.java

```diff
@@ -58,6 +64,14 @@
         }
         return errors.isEmpty() ? Optional.empty() : Optional.of(errors);
     }
+    
+    public Optional<List<String>> isValidEnumValue(JsonNode payload, Class enumClass, String field) {
+        String value = payload.get(field).asText();
+        if (EnumSet.allOf(enumClass).stream().noneMatch(constant -> constant.toString().equals(value))) {
```

Comment:
We could have this method take an `EnumSet` instead of the enum `Class` object and avoid having to recreate it i.e. make the method signature `isValidEnumValue(JsonNode payload, EnumSet<?> enumClass, String field)`